curriculum/challenges/english/01-responsive-web-design/css-grid/create-flexible-layouts-using-auto-fill.english.md

```diff
@@ -8,7 +8,7 @@ videoUrl: 'https://scrimba.com/p/pByETK/cmzdycW'
 ## Description
 <section id='description'>
 The repeat function comes with an option called <dfn>auto-fill</dfn>. This allows you to automatically insert as many rows or columns of your desired size as possible depending on the size of the container. You can create flexible layouts when combining <code>auto-fill</code> with <code>minmax</code>.
-In the preview, <code>grid-template-columns</code> is set to
+In the preview, <code>grid-template-columns</code> for <code>.container2</code> is set to
 <blockquote>repeat(auto-fill, minmax(60px, 1fr));</blockquote>
 When the container changes size, this setup keeps inserting 60px columns and stretching them until it can insert another one.
```

Comment:
```suggestion
When  <code>.container2</code> changes size, this setup keeps inserting 60px columns and stretching them until it can insert another one.
```